### PR TITLE
Improve countdown message

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso @ahmedalsudani please review

We'll need some CSS help for this

### CODE REVIEW

**Comment by ahmedalsudani** [comment]
:+1:

File: installer/ui/ipc.js [hunk 1]
```diff
@@ -76,13 +76,21 @@ ipc.on("start-unattended", ()=> {
...
 });
 
-ipc.on("start-unattended-countdown", ()=> {
+ipc.on("start-unattended-countdown", (evt, detail)=> {
   var currentSlide = document.querySelector(".container.slide.active"),
-  unattendedCountdownSlide = document.querySelector("#unattendedCountdown");
+  unattendedCountdownSlide = document.querySelector("#unattendedCountdown"),
+  playerInstallationHeader = document.querySelector("#playerInstallationHeader"),
+  restartBlocks = document.querySelectorAll(".restartBlock"),
+  isWindows10 = detail.isWindows && detail.version === "10";
```
Comment (tejohnso) [inline installer/ui/ipc.js]: @fjvallarino If I recall correctly the matching comparison for killExplore would be detail.isWindows && detail.version !== "7".

Comment (fjvallarino) [inline installer/ui/ipc.js]: @tejohnso Good point. I'll change it to be consistent.

**Comment by fjvallarino** [comment]
@tejohnso @ahmedalsudani updated

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**